content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+Laman ini adalah sebuah ikhtisar dari dokumentasi React dan sumber-sumber terkait.
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** adalah sebuah *library* JavaScript untuk membangun antarmuka pengguna (*user interface*). Pelajari apa sebenarnya React itu di [beranda kami](/) atau [di dalam tutorial](/tutorial/tutorial.html).
 
 ---
 
-- [Try React](#try-react)
-- [Learn React](#learn-react)
-- [Staying Informed](#staying-informed)
-- [Versioned Documentation](#versioned-documentation)
-- [Something Missing?](#something-missing)
+- [Coba React](#coba-react)
```

Comment:
I don't think we should translate the hyperlink. All other languages translation left it as it is. This should still be `#try-react`. Reference https://es.reactjs.org/docs/getting-started.html#try-react